prow/jobs/custom/autodeploy-prow.yaml

```diff
@@ -18,8 +18,10 @@ postsubmits:
     cluster: prow-trusted
     run_if_changed: '^prow/cluster/'
     decorate: true
+    decoration_config: # will remove this temp override
```

Comment:
Assuming it is intentional that this is opposite from prow/config.yaml ?

Reply:
If that's the plan, shouldn't `gcs-upload` be replaced as `test-account` here?